components/script/dom/readablestreamdefaultreader.rs

```diff
@@ -29,18 +34,47 @@ pub enum ReadRequest {
 
 impl ReadRequest {
     /// <https://streams.spec.whatwg.org/#read-request-chunk-steps>
+    #[allow(unsafe_code)]
+    #[allow(crown::unrooted_must_root)]
```

Comment:
Why is this annotation required?

Reply:
I was able to remove it by avoiding to assign the read result to a temporary variable.

Reply:
I was able to remove it by avoiding to assign the read result to a temporary variable.